extensions/data-transfer/portability-data-transfer-flickr/src/main/java/org/datatransferproject/datatransfer/flickr/photos/FlickrPhotosExporter.java

```diff
@@ -136,6 +144,37 @@ static String toMimeType(String flickrFormat) {
     }
   }
 
+  private ExportResult<PhotosContainerResource> exportPhotosContainer(
+      PhotosContainerResource container) {
+    ImmutableList.Builder<PhotoAlbum> albumBuilder = ImmutableList.builder();
+    ImmutableList.Builder<PhotoModel> photosBuilder = ImmutableList.builder();
+    List<IdOnlyContainerResource> subResources = new ArrayList<>();
+
+    try {
+      for (PhotoAlbum album : container.getAlbums()) {
+        Photoset photoset = photosetsInterface.getInfo(album.getId());
+        // Saving data to the album allows the target service to recreate the album structure
+        albumBuilder.add(
+            new PhotoAlbum(photoset.getId(), photoset.getTitle(), photoset.getDescription()));
+        // Adding subresources tells the framework to recall export to get all the photos
+        subResources.add(new IdOnlyContainerResource(photoset.getId()));
+      }
+
+      for (PhotoModel photo : container.getPhotos()) {
+        Photo p = photosInterface.getInfo(photo.getDataId(), null);
+        photosBuilder.add(toCommonPhoto(p, null));
+      }
+    } catch (FlickrException e) {
+      return new ExportResult<>(e);
```

Comment:
Any additional logging we want to try here?

Reply:
I followed the same thing we were doing with the individual getAlbums() and getPhotos() calls. The exception would be logged up a level (in the executor)